src/Curiosity/curiosity.tsx

```diff
@@ -48,18 +48,26 @@ function Curiosity() {
 
     return (
         <div>
+
+            <div className='toppagebreak'></div>
```

Comment:
Let's use the convention we've used else where especially on this page to call it top-page-break and same for the other ones